src/api/procedures/__tests__/consumeJoinIdentityAuthorization.ts

```diff
@@ -40,6 +41,7 @@ describe('consumeJoinIdentityAuthorization procedure', () => {
     rawFalse = dsMockUtils.createMockBool(false);
 
     sinon.stub(utilsConversionModule, 'addressToKey');
+    sinon.stub(utilsInternalModule, 'assertFormatValid');
```

Comment:
Instead of doing this, you should change all instantiations of Account in these tests to `entityMockUtils.getAccountInstance`